conan/cli/formatters/graph/graph.py

```diff
@@ -73,7 +73,7 @@ def _build_graph(self):
                 dst = _node_map[node_to]
                 edges.append((src, dst))
 
-        return _node_map.values(), edges
+        return _node_map.values(), edges, _node_map
```

Comment:
Could probably simplify this to only return the node_map and the edges, will do so later today